src/OpenIddict.Abstractions/OpenIddictConstants.cs

```diff
@@ -365,6 +365,15 @@ public static class Parameters
         public const string VerificationUriComplete = "verification_uri_complete";
     }
 
+    public static class ClaimsParameters
```

Comment:
`ClaimRequestMembers` may be a better name.

Reply:
Considering they are called `ClaimParameter` in the spec, I went with that name.
I don't think `ClaimRequestMembers` is a good match though, perhaps `ClaimRequestParameters` or `RequestClaimParameters` or `ClaimRequestParametersMembers`? I still think the `ClaimsParameters` is the best though.

Reply:
alternatively, a split design:
empty `ClaimRequest` and `ClaimRequest.RequestTypes` with `userinfo` and `id_token`, and `ClaimRequest.Members` with the rest.

Reply:
> Considering they are called `ClaimParameter` in the spec, I went with that name.

I'm quite familiar with the spec but I re-read it just to be sure and I don't think it's true. The only parameter mentioned in the 5.5 section is named `claims`: it's the JSON object that is base64url-encoded and contains two top-level `id_token` and `userinfo` nodes. These nodes present in the claims request are simply called "members":

![image](https://github.com/openiddict/openiddict-core/assets/6998306/5a65cb66-2732-4288-959f-51827a115b78)

Reply:
The url is `openid-connect-core-1_0.html#ClaimsParameter`.
In any case, what is your final preferred naming scheme and I shall update :)

Reply:
> The url is `openid-connect-core-1_0.html#ClaimsParameter`. In any case, what is your final preferred naming scheme and I shall update :)

Well, the anchor points to the section title, that clearly says "`claims` Request Parameter": only `claims` is technically defined as a parameter, not the JSON nodes defined in the section itself.

Let's go with `ClaimRequestMembers`. It's not a great name, but it's the most accurate one.

Reply:
Ok. I felt if you wanted to very strictly keep to the nomenclature used in the actual text of the spec, that we should also split it into `Individual` and `TopLevel`

Reply:
Well, it's not wrong, but I'm not so sure it's worth being that specific (it makes names longer). Let's just flatten the class (please also remove the comments, the other classes don't have them).